ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
```

Comment:
The `dbus` module is not available on a ZeroPhone, even after `pydbus` is installed